ports/nanopb/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "nanopb",
-  "version-semver": "0.4.5",
+  "version-semver": "0.4.7",
   "description": "A small code-size Protocol Buffers implementation in ANSI C.",
   "homepage": "https://jpa.kapsi.fi/nanopb/",
```

```suggestion
  "homepage": "https://jpa.kapsi.fi/nanopb/",
  "license": "Zlib",
```

A `license` is required.